offsetfetch.go

```diff
@@ -81,10 +81,20 @@ func (c *Client) OffsetFetch(ctx context.Context, req *OffsetFetchRequest) (*Off
 		})
 	}
 
-	m, err := c.roundTrip(ctx, req.Addr, &offsetfetch.Request{
-		GroupID: req.GroupID,
-		Topics:  topics,
-	})
+	var offsetFetchReq *offsetfetch.Request
+
+	if len(req.Topics) < 1 {
```

Comment:
Can you add a comment here that topics can be null and kafka will return info on all topics as per spec